server/config.go

```diff
@@ -35,13 +36,19 @@ type Config struct {
 	Options Options            `yaml:"options"`
 	Logging LoggingConfig      `yaml:"logging"`
 	Datadog datadog.Config     `yaml:"datadog"`
+	Cache   CacheConfig        `yaml:"cache"`
 }
 
 type LoggingConfig struct {
 	Level string `yaml:"level"`
 	Text  bool   `yaml:"text"`
 }
 
+type CacheConfig struct {
```

Comment:
Do you think we should instead set this config struct in the githubapp library?

Reply:
Let's leave this here for now, with the reasoning that we're not recommending any particular cache implementation in `go-githubapp`. If we end up using it a lot, I could see moving the config up a level and adding something like `WithClientLRUCaching(config)`, but I don't think the config struct alone is valuable without something in the library that uses it.